doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1786,13 +1786,13 @@ Tactic notations allow customizing the syntax of tactics.
 
       * - ``reference``
         - :token:`qualid`
-        - a global reference of term
-        - :tacn:`unfold`
+        - a qualified identifier
+        - the head of an |Ltac| call
```

Comment:
"head of an |Ltac| call" meaning the name of an Ltac-defined tactic (sort of like the name of a function)?

If "name of an |Ltac|-defined tactic" doesn't work, is there another succinct way to express this?  I've not seen a definition of "head" in the doc, so for me and maybe others it is not clear.  @Zimmi48 any thoughts?

Reply:
Right, "head" is a bit technical term in lambda-calculus. The formulation "name of an |Ltac|-defined tactic" is much better. I changed it.